settings.gradle

```diff
@@ -29,3 +25,21 @@ include 'test-suite-kotlin'
 include 'grpc-health'
 include 'grpc-opentracing'
 
+enableFeaturePreview 'TYPESAFE_PROJECT_ACCESSORS'
```

Comment:
Change all the build.gradle files to use this feature; e.g. instead of `api project(":grpc-annotation")` use `api projects.grpcAnnotation`